File: Electronics_Shop_17.Model/Requests/AddPerson.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Electronics_Shop_17.Model.Requests
{
    public class AddPerson
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime DateOfBirth { get; set; }


    }
}

File: angular17_web/src/app/models/person/add.person.ts
export class AddPerson {
  firstName!: string;
  lastName!: string;
  dateOfBirth!: Date;

  constructor(data?: any) {
    if (!data) return;

    this.firstName = data.firstName;
    this.lastName = data.lastName;
    this.dateOfBirth = data.dateOfBirth ? new Date(data.dateOfBirth) : undefined as any;
  }
}
